### ISSUE TITLE
Machine 

### ISSUE THREAD

**Issue by Matej1006** [gnea/grbl#583]
Is that machine ok to use

**Comment by Fusseldieb** [comment]
GRBL is mainly made for CNC and/or Lasers. "Bagat jadranka" turns out to be a sewing machine, upon a quick Google search.

A sewing machine hasn't a X or Y position, only a Z. It wouldn't make any sense.

But if you somehow manage to get motors to move the material in the X/Y direction, you just need to hook up Z also and you're good to go.

"All" you need is a Arduino, Stepper Motors, Stepper Drivers, a power supply and a lot of patience. 

Basically, it doesn't matter if it's a sewing machine, a CNC, a Laser, a 3D printer, a microwave or a cat. All it takes is G-Code and turns it into motor movement.
